build.gradle

```diff
@@ -112,19 +112,19 @@ subprojects { project ->
             java {
                 googleJavaFormat()
             }
-            groovyGradle {
-                target '**/*.gradle'
-                greclipse()
-                indentWithSpaces(4)
-                trimTrailingWhitespace()
-                endWithNewline()
-            }
         }
     }
 }
 
 spotless {
     predeclareDeps()
+    groovyGradle {
+        target '**/*.gradle'
+        greclipse()
+        indentWithSpaces(4)
+        trimTrailingWhitespace()
+        endWithNewline()
+    }
```

Comment:
Moving these lines from their previous location is the only real change in this PR.  The remaining changes are just reformatting